python/src/trezorlib/cardano.py

```diff
@@ -172,19 +192,26 @@ def parse_output(output) -> OutputWithAssetGroups:
     )
 
 
-def _parse_token_bundle(token_bundle) -> List[AssetGroupWithTokens]:
+def _parse_token_bundle(token_bundle, is_mint: bool) -> List[AssetGroupWithTokens]:
+    error_message = (
+        INVALID_MINT_TOKEN_BUNDLE_ENTRY
+        if is_mint
+        else INVALID_OUTPUT_TOKEN_BUNDLE_ENTRY
+    )
```

Comment:
I really dislike the formatting of these ternary operators. If this is what Trezor likes most, then OK, but I find an if-else statement much clearer.

Reply:
https://github.com/vacuumlabs/trezor-firmware/pull/66/commits/ca9a89b2b753fbf0d744fb3bbec1335081ed6089#diff-aa355fcce844deffafbd696c1d414eb1c0198b0af2726710c7b8d8f7fda577cdR202